website/docs/guides/orchestration/webhooks/zapier-ms-teams.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Post to Microsoft Teams when a dbt Cloud job finishes"
+id: webhooks-guide-zapier-ms-teams
+slug: zapier-ms-teams
+description: Use Zapier and webhooks to send info to Microsoft Teams
+---
+
+This guide assumes familiarity with:
+- dbt Cloud webhooks
+- Zapier
+
+## What does this integration do? 
+When a dbt Cloud job completes:
+ - receive a webhook notification in Zapier
+ - extract the results from the dbt Cloud admin API 
+ - post a summary to a Microsoft Teams channel
```

Comment:
Callout box here (or somewhere else) showing how you can do this in slack in case people find this and are looking for slack